src/EnergyPlus/DisplacementVentMgr.cc

```diff
@@ -516,7 +516,8 @@ void CalcUCSDDV(EnergyPlusData &state, int const ZoneNum) // Which Zonenum
     // Using/Aliasing
     using namespace DataEnvironment;
     using namespace DataHeatBalance;
-    auto &TimeStepSys = state.dataHVACGlobal->TimeStepSys;
+    Real64 TimeStepSys = state.dataHVACGlobal->TimeStepSys;
+    Real64 TimeStepSysSec = TimeStepSys * DataGlobalConstants::SecInHour;
```

Comment:
TimeStepSys is only used here. Wouldn't it be better to use:

    Real64 TimeStepSysSec = state.dataHVACGlobal->TimeStepSys * DataGlobalConstants::SecInHour;

Reply:
Nope, I see it at line 581.

Reply:
A value shortcut is always fine even if it is only used once, because: i) this is essentially what the compiler does itself (it has to evaluate every expression piece by piece so each sub-expression essentially gets a register shortcut) and ii) you are not adding a pointer or reference for the compiler to keep track of and which may interfere with other optimizations.